### ISSUE TITLE
Keybindings: Add edit shortcuts

### ISSUE THREAD

**Issue by stormpython** [nteract/archived-desktop-app#4103]
**Is your feature request related to a problem? Please describe.**
Add `Edit` shortcuts to nteract.

- [ ]  redo => `ctrl+shift+z`
- [ ] undo => `ctrl+z`
~Paste Cell(s) Below => `ctrl+shift+v`~

This issue is a child of #4101.

**Describe the solution you'd like**
- [x] ~*Before this can be solved, #4054 needs to be merged.*~

For information on how to add shortcuts to nteract, please see [4102](https://github.com/nteract/nteract/issues/4102#issuecomment-457790015). See the list of [nteract shortcut-key](https://docs.nteract.io/#/desktop/shortcut-keys).

**Comment by stormpython** [comment]
@kelleyblackmore I think this would be a good one to start with. Please let me know if you have any problems or questions.

**Comment by kelleyblackmore** [comment]
Sounds good! sorry about the delayed response

**Comment by stormpython** [comment]
@kelleyblackmore, feel free to take a look at either PR #4138 or #4139 for an example of how to hook up these keybindings.

**Comment by stormpython** [comment]
@kelleyblackmore, this is actually a much bigger project than I previously thought. There aren't any `actions` or `reducers` for **undo** and **redo**. I think we might want to table this for a bit until we have a better idea of what is required here.

**Comment by captainsafia** [comment]
Following up on this thread.

The undo functionality that we currently have is limited to the undo/redo within the CodeMirror editor itself. In the past, there were attempts to add undo/redo actions to our application but those were reverted (ref: https://github.com/nteract/nteract/pull/510, https://github.com/nteract/nteract/pull/627).

If we want to unblock this work, we can make the keyboard shortcuts for Undo and Redo have parity with current functionality (undo and redo in CodeMirror). We can also table that work in the shortcuts work for now.

**Comment by captainsafia** [comment]
We have undo available now so we can configure a keyboard shortcut for that. I'm closing this in favor of https://github.com/nteract/nteract/issues/4781 where we can track all our keyboard shortcuts-related issues.
